cmd/calyptia/main.go

```diff
@@ -77,6 +78,14 @@ func newCmd(ctx context.Context) *cobra.Command {
 		config.projectToken = token
 		config.projectID = projectID
 	})
+
```

Comment:
This should be moved to PreRun of the root command. So we don't have to repeat this